ddtrace/context.py

```diff
@@ -123,11 +130,81 @@ def sampling_priority(self, value):
     @property
     def _traceparent(self):
         # type: () -> str
-        if self.trace_id is None or self.span_id is None:
+        tp = self._meta.get(_TRACEPARENT_KEY)
+        dd_trace_id = self.trace_id
+        span_id = self.span_id
```

Comment:
```suggestion
        dd_trace_id = self.trace_id
        span_id = self.span_id
```

We can remove these variables. They just add complexity. Let's call `self.span_id` and `self.trace_id` directly